Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -24,6 +25,8 @@ fileprivate let placesFetchMonitorIdentifier = "PlaceFetchRadiusMonitor"
 
 class PlaceCarouselViewController: UIViewController {
 
+    lazy var placesWithEvents = Set<Place>()
```

Comment:
I noticed that we never assign anything to this - it's always the empty set. Is this needed or is there an assignment missing? If we don't need it we can probably clean up a lot of the changes in the dataSource that was made to accommodate this set.